draft-ietf-quic-transport.md

```diff
@@ -3700,10 +3742,10 @@ FIN bit.
 A sender bundles one or more frames in a QUIC packet (see {{frames}}).
 
 A sender SHOULD minimize per-packet bandwidth and computational costs by
-bundling as many frames as possible within a QUIC packet.  A sender MAY wait for
-a short period of time to bundle multiple frames before sending a packet that is
+bundling multiple frames within a QUIC packet.  A sender MAY wait for
+a short period of time for additional frames before sending a packet that is
 not maximally packed, to avoid sending out large numbers of small packets.  An
-implementation may use knowledge about application sending behavior or
+implementation MAY use knowledge about application sending behavior or
```

Comment:
I don't know that this needs to be normative.  If you want to avoid the lower-case may, perhaps "can"?

Reply:
Was not sure here. Changed to can for unambiguity.